index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Women In Tech summit</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <script src="https://kit.fontawesome.com/953df531f5.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <!-- NAV -->
+  <main class= "d-flex flex-column mt-5">
+    <nav class="bg-dark main-bar">
+      <div class="main-nav d-flex justify-content-end align-items-center">
+        <a href="#"><i class="fa-brands fa-twitter text-light mx-2 mainnav-link"></i></a>
+        <a href="#"><i class="fa-brands fa-facebook text-light mx-2 mainnav-link"></i></a>
+        <a href="#" class="mx-2 nav-link text-light mainnav-link">English</a>
+        <a href="#" class="mx-2 nav-link text-light mainnav-link">My Page</a>
+        <a href="#" class="mx-2 nav-link text-light mainnav-link">Logout</a>
+      </div>
+    </nav>
+  
+  <section class= "bg-white nav-sub">
+    <div class="container-fluid d-flex menu-bar">
+      <a href="" class="nav-brand px-2 h5">Women tech<br> Summit 2022</a>
+      <ul class=" nav-ul align-items-center justify-content-center">
+        <li class="item mx-1"><a href="./about.html" class="nav-link h3">About</a></li>
+        <li class="item mx-1 "><a href="#" class="nav-link h3">Resources</a></li>
+        <li class="item mx-1 "><a href="#" class="nav-link h3">Awards</a></li>
+        <li class="item mx-1 "><a href="#" class="nav-link h3">Join</a></li>
+        <li class="item mx-1 "><a href="#" class="nav-link h3">Sponsor</a></li>
+        <li class="item mx-1  "><a href="#" class="nav-link h3">News</a></li>
+
+      </ul> 
+      <div class="hambur my-2">
+        <div class="lin lin1"></div>
+        <div class="lin lin2"></div>
+        <div class="lin lin3"></div>
+      </div> 
+    </div>
+  </section>
+</main>
+<!-- END OF NAV -->
+<!-- Main Section -->
+  <section class="header">
+    <main class="img-cover">
+     <h3 class="h3 text-white-50 container text-center pt-5 pb-1 mt-5">"Connect with like minds"</h3>
+     <div class="headline container py-3 mt-3">
+       <h1 class="h1 headtext">Women In Tech General Summit 2022</h1>
+       <div class="sep mx-3 my-4"></div>
+       <h6 class="venue text-primary mb-3">Dec 12th, 2022, 8pm | Zoom </h6>
+       <a href="#" class="learn py-2 px-2">Learn more</a>
+      </div>
+     <p class="theme h4 pt-5 px-2 text-center">Unite 100 000 Women in Tech to Foster Innovation with Purpose and Impact</p>
+
```

Comment:
- [ ] Comparing your headline section with the one in the given design, I can see that there is a big difference. Please take a look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/198257057-4a114a37-dfd8-472b-b4ce-2cb2cf5a27ca.png)
**Given design**
![image](https://user-images.githubusercontent.com/90222110/198257184-df6dae36-3b4b-4e63-be6e-549d92086f8e.png)

From the design guidelines, you are expected to follow the alignment and layout exactly as it is in the design. Also for the texts, you should use the same color as in the design. **Your texts are aligned in the center, while on the design given, they are aligned on the left side. The headline text on your design is not very big and it is white while on the given design the headline text is big and orange in color.** 
These are a few example differences you can see, therefore I would advise you to improve your styles in order to match the given design very well. this showcase your ability to parse designs 😎